api/v1alpha1/openapi.yaml

```diff
@@ -858,18 +858,18 @@ paths:
       description: get the full specification for the specified device
       operationId: getRenderedDeviceSpec
       parameters:
-        - name: name
-          in: path
-          description: Name of the device
-          required: true
-          schema:
-            type: string
```

Comment:
Oh I must move this back to the original place, I was hopping to un-trigger the openapi bug for the import paths...